crates/shared/src/http_solver/model.rs

```diff
@@ -477,6 +477,9 @@ pub enum SolverRejectionReason {
 
     /// It is expected for a score to be less or equal to the objective value.
     ScoreHigherThanObjective,
+
+    /// Solver balance too low to cover the execution costs.
+    InsufficientBalance(String),
```

Comment:
Not a fan of passing a string that is constant (we always set it to the same value). Can we rename this type to make it more obvious (e.g. `SolverAccountInsufficientBalance`)

Reply:
Killed the message and renamed variant which should be clear enough.